packages/jaeger-ui/src/components/common/TraceIdDisplayLength.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2023 The Jaeger Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import { getConfigValue } from '../../utils/config/get-config';
+import './TraceIdDisplayLength.css';
+
+type Props = {
+  traceId: string;
+  className?: string;
+};
+
+export function TraceIdDisplayLength({ traceId, className = '' }: Props) {
```

Comment:
```suggestion
export function TraceId({ traceId, className = '' }: Props) {
```
"DisplayLength" is implementation detail irrelevant to the user.